Lib/gftools/fix.py

```diff
@@ -138,7 +161,11 @@ def fix_hinted_font(ttFont):
     Args:
         ttFont: a TTFont instance
     """
+    if not 'fpgm' in ttFont:
+        return False, ["Skipping. Font is not hinted."]
```

Comment:
Adding this condition breaks test `Lib/gftools/tests/test_fix.py::test_fix_hinted_font`.

To fix, you can either update the test or turn the unhinted test font into a mock hinted font by inserting an `fpgm` table e.g `static_font['fpgm'] = newTable("fpgm")` on L62. The latter option seems easiest to me, otherwise we may need to include another test font which has been run through ttfa or select one which is hinted.